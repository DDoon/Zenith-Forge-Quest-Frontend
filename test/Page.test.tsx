import { render, screen } from '@testing-library/react'
import Page from '../app/page'

describe('<Page />', () => {
  it('test', () => {
    const { container } = render(<Page />)

    const home = screen.getByText('테스트')

    expect(home).toBeInTheDocument()
    expect(container).toMatchSnapshot()
  })
})
