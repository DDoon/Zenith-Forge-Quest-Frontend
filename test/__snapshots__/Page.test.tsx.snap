// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Page /> test 1`] = `
<div>
  <main>
    <div>
      테스트
    </div>
    <a
      href="/test"
    >
      테스트페이지 이동
    </a>
  </main>
</div>
`;
